lib/utils.ts

```diff
@@ -1,6 +1,19 @@
-import { type ClassValue, clsx } from "clsx";
-import { twMerge } from "tailwind-merge";
+import { extendedTheme } from "@/tailwind.config"
+import { type ClassValue, clsx } from "clsx"
+import { extendTailwindMerge } from "tailwind-merge"
+
+const customTwMerge = extendTailwindMerge({
+  extend: {
+    classGroups: {
+      'font-size': [
+        {
+          text: Object.keys(extendedTheme.fontSize),
+        },
+      ],
+    },
+  },
+})
```

Comment:
I think we should introduce a namespace for global functionality like this. 
I suggest: `lib/helpers`
And this file could then be placed in there like: `lib/helpers/tailwind.ts`

Reply:
I am down for `lib/helpers/` (I'm already using it in one of my PRs 😛